cstool/cstool_hppa.c

```diff
@@ -71,10 +65,11 @@ void print_insn_detail_hppa(csh handle, cs_insn *ins)
                     i, cs_reg_name(handle, op->mem.base));
             break;
         case HPPA_OP_TARGET:
+            printf("\t\toperands[%u].type: ", i);
             if (op->imm >= 0x8000000000000000)
-                printf("TARGET: -0x%lx\n", -op->imm);
+                printf("TARGET = -0x%lx\n", -op->imm);
```

Comment:
For the string formatting please use the `PRI...` macros in `platform.h`, so it is compatible to every OS.